cypress/support/cypress-commands/lifecycle.js

```diff
@@ -254,8 +259,6 @@ Cypress.Commands.add('validateLifecycleHistoryAndEvents', (state, appId) => {
             );
           }
```

Comment:
why is this condition removed ?

Reply:
updated as per main branch and added only the required changes after discussing with Nandana